### PR TITLE
Updated link to yarn installation guide

### PR DESCRIPTION
**Pull request by chagai95**
The old link was broken.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Development
 Ensure you have the latest LTS version of Node.js installed.
 
 Using `yarn` instead of `npm` is recommended. Please see the Yarn [install
-guide](https://yarnpkg.com/docs/install/) if you do not have it already.
+guide](https://yarnpkg.com/getting-started/install) if you do not have it already.
```
Comment (jryans) [inline README.md:137]: Hmm, would https://classic.yarnpkg.com/docs/install be simpler to understand perhaps? (We aren't using Yarn 2 yet.)

Comment (chagai95) [reply README.md:137]: Wasn't aware of that, looks much better! Could you add that you aren't using yarn 2 somewhere as well?

Comment (jryans) [reply README.md:137]: We should do that, yes. Perhaps you could do it as part of your change here? Maybe something like edit "Please see the Yarn install guide ..." to "Yarn 1", and then add a new sentence such as "This project has not yet been migrated to Yarn 2, so please ensure `yarn --version` shows a version from the 1.x series."

**Comment by jryans** [comment]
Replaced by https://github.com/matrix-org/matrix-react-sdk/pull/4589

**Pull request closed without merging**